recipes/fastjet/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "fastjet" %}
+{% set version = "3.3.4.0rc8" %}
```

Comment:
Do you plan to change this to a release version before the release?

Reply:
The rc's will continue for a few more until a full release version. @jpivarski and @aryan26roy can tell you more about that path.

I'd like to have it in conda from the release candidates since we're already using the package in conda environments via pip (and it's a benefit to have everything in conda).

Reply:
Then please use a `dev/rc` channels as we do in https://github.com/conda-forge/lief-feedstock/blob/dev/recipe/conda_build_config.yaml to mark them as such. The version numbers in `conda` don't differentiate between release and pre-release. Therefore pre-release should be put into labels.